extension/jmxmetricsextension/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/extension/jmxmetricsextension
+
+go 1.14
+
+require (
+	github.com/stretchr/testify v1.6.1
+	go.opentelemetry.io/collector v0.8.0
```

Comment:
Please use the same version as everywhere.

Reply:
updated